core/services/run_queue.go

```diff
@@ -12,11 +13,11 @@ import (
 )
 
 var (
-	numberRunsQueued = promauto.NewCounter(prometheus.CounterOpts{
+	promNumberRunsQueued = promauto.NewCounter(prometheus.CounterOpts{
 		Name: "run_queue_runs_queued",
 		Help: "The total number of runs that have been queued",
 	})
-	numberRunQueueWorkers = promauto.NewGauge(prometheus.GaugeOpts{
+	promNumberRunQueueWorkers = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
I did find this confusing for a moment, but upon inspection, Prometheus's `Gauge` turns out to represent a monotonically increasing integer.  Knowing this ought to be enough to infer the behavior.

Reply:
Hmm, the [docs in `master`](https://github.com/prometheus/client_golang/blob/master/prometheus/gauge.go#L24-L29) say otherwise:

```golang
// Gauge is a Metric that represents a single numerical value that can
// arbitrarily go up and down.
//
// A Gauge is typically used for measured values like temperatures or current
// memory usage, but also "counts" that can go up and down, like the number of
// running goroutines.
```